packages/app/src/providers/Provider.tsx

```diff
@@ -68,7 +68,10 @@ function Provider(props: Props) {
   const filepathMembers = api.filepath.split('/');
 
   const attrValuesStore = useMemo(() => {
-    const store = createFetchStore(api.getAttrValues.bind(api));
+    const store = createFetchStore(api.getAttrValues.bind(api), {
+      type: 'Map',
+      areEqual: (a, b) => a.path === b.path,
```

Comment:
I realised when removing the internal cache from `h5grove-api` that the store's cache wasn't working properly.